roles/wazo_auth/tasks/main.yml

```diff
@@ -0,0 +1,135 @@
+  when:
+    - engine_api_db_admin_password is defined
+    - engine_api_db_admin_user is defined
+    - engine_api_db_auth_name is defined
+    - engine_api_db_auth_password is defined
+    - engine_api_db_auth_user is defined
+    - engine_api_db_confd_name is defined
+    - engine_api_db_confd_password is defined
+    - engine_api_db_confd_user is defined
+    - engine_api_db_host is defined
+    - engine_api_db_port is defined
+    - runtime
+
+- name: Copy wazo-auth database preconfiguration template
+  copy:
+    src: templates/wazo-auth.cfg.j2
+    dest: /etc/wazo-auth.cfg.j2
+  when: not runtime
+
+- name: Reset database options for wazo-auth package
+  shell: "echo PURGE | debconf-communicate wazo-auth"
+  when: runtime
+
+- name: Reconfigure database options for wazo-auth package
+  command: debconf-set-selections "{{ wazo_auth_debconf_file.path }}"
+  when: runtime
+
+- name: Initialize wazo-auth database
+  command: dpkg-reconfigure wazo-auth --frontend noninteractive
+  when: runtime
+
+- name: Install wazo-auth-keys
+  apt:
+    name: wazo-auth-keys
+    state: latest
+
+- name: Create service users with wazo-auth-keys
+  command: wazo-auth-keys service update
+  when: runtime
+
+- name: Create default main database preconfiguration
+  tempfile:
+    state: file
+    prefix: xivo_managedb_debconf_file
+  register: xivo_managedb_debconf_file
+  when: runtime
+
+- name: Copy main database preconfiguration
+  template:
+    src: templates/xivo-manage-db.cfg.j2
+    dest: "{{ xivo_managedb_debconf_file.path }}"
+  when:
+    - engine_api_db_admin_password is defined
+    - engine_api_db_admin_user is defined
+    - engine_api_db_confd_name is defined
+    - engine_api_db_confd_password is defined
+    - engine_api_db_confd_user is defined
+    - engine_api_db_host is defined
+    - engine_api_db_port is defined
+    - runtime
+
+- name: Copy main database preconfiguration template
+  copy:
+    src: templates/xivo-manage-db.cfg.j2
+    dest: /etc/xivo-manage-db.cfg.j2
+  when: not runtime
+
+- name: Reset main database options for wazo-auth package
+  shell: "echo PURGE | debconf-communicate xivo-manage-db"
+  when: runtime
+
+- name: Reconfigure main database options
+  command: debconf-set-selections "{{ xivo_managedb_debconf_file.path }}"
+  when: runtime
+
+- name: Initialize main database
+  command: dpkg-reconfigure xivo-manage-db --frontend noninteractive
+  when: runtime
+
+- name: Fix Twisted dropin.cache
+  command: twistd --help-reactors
+  when: runtime
+
+- name: Start Wazo services
+  service:
+    name: "{{ item }}"
+    state: restarted
+  loop:
+    - wazo-agid
+    - wazo-amid
+    - wazo-confd
+    - wazo-confgend
+    - wazo-calld
+    - wazo-provd
+    - wazo-websocketd
+  when: runtime
+
+- name: Deconfigure wazo-auth package
+  debconf:
+    name: wazo-auth
+    question: "{{ item }}"
+    vtype: boolean
+    value: false
+  loop:
+    - wazo-auth/bootstrap-skip
+    - wazo-auth/db-skip
+  when: not runtime
```

Comment:
Lines 60 to 135: Those tasks are not related to wazo-auth. They shoud go either in their own role, or back to engine-api-init.